egs2/TEMPLATE/asr1/db.sh

```diff
@@ -47,13 +42,12 @@ LIBRILIGHT_LIMITED=
 FSC=
 SLURP=
 VOXCELEB=
-MEDIASPEECH=downloads
 MINI_LIBRISPEECH=downloads
 MISP2021=
 LIBRIMIX=downloads
 LIBRITTS=
 LJSPEECH=downloads
-MUSAN=
+qasr_tts=downloads
```

Comment:
To keep the convention of capital letter, maybe use `QASR_TTS` instead of qasr_tts